test/services/MeasureService.test.ts

```diff
@@ -0,0 +1,45 @@
+import { initialize, Server } from '@projecttacoma/node-fhir-server-core';
+import { serverConfig } from '../../src/config/serverConfig';
+import { cleanUpDb, testSetup } from '../utils';
+import supertest from 'supertest';
+
+let server: Server;
+
+const Measure: fhir4.Measure = { resourceType: 'Measure', id: 'measure123', status: 'active' };
+
+describe('MeasureService', () => {
+  beforeAll(async () => {
+    const config = serverConfig;
+    server = initialize(config);
+
+    await testSetup([Measure]);
+  });
+
+  describe('searchById', () => {
+    it('test searchById with correctHeaders and the id should be in database returns 200', async () => {
+      await supertest(server.app)
+        .get('/4_0_1/Measure/measure123')
+        .set('Accept', 'application/json+fhir')
+        .set('content-type', 'application/json+fhir')
```

Comment:
Nitpicking here, but this content-type header actually does nothing.

It matters in the request when we are specifying the content-type of the request body, which is impossible in GET requests.

It will matter if we invoke post requests with FHIR resources as the body


```suggestion
```